### ISSUE TITLE
Research different ML algorithms used to classify cancerous skin lesions

### ISSUE THREAD

**Issue by sagarVL** [sagarVL/TUHH_ISM_tumor_detection#11]
Find papers and implementations (maybe some on GitHub) which deal with the classification or identification of cancerous skin lesions. Create a comparative study of all the found literature by differentiating each implementation based on the training, testing, and /or validation accuracy.

**Comment by Vignesh-1997** [comment]
[Comparison](https://www.researchgate.net/publication/321937436_Skin_Lesion_Classification_using_Machine_Learning_Algorithms)

**Comment by sagarVL** [comment]
@Vignesh-1997 restrict your research to classification(classical ML) algorithms like SVM, logistic regression, etc. I suggest not to venture into the 'neural networks for classification area' yet as that would be the second task. Also, can you create a small PPT, briefly describing what each algorithm does, do not dwell too much into the maths but stay in the algorithm level. In the PPT, you can talk about the results at the end too. So, overall something you would do when you submit a literature survey!

**Comment by sagarVL** [comment]
@Vignesh-1997 can you comment on the recent issue which I created with links to papers and implementation you found regarding SVM and logistic regression?
